server/src/main/resources/application.conf

```diff
@@ -150,3 +150,22 @@ crunch {
     raw-data-path = ${?RAW_SPLITS_DATA_PATH}
   }
 }
+
+feeds {
+  gatwick {
+    live {
+      azure {
+        namespace = ""
```

Comment:
if you use the option syntax  _"${?varname}"_ , I don't think you need a default blank value here.

Reply:
(it's been done incorrectly elsewhere in this file)